tests/models/roberta/test_modeling_roberta.py

```diff
@@ -579,23 +578,3 @@ def test_inference_classification_head(self):
         # expected_tensor = roberta.predict("mnli", input_ids, return_logits=True).detach()
 
         self.assertTrue(torch.allclose(output, expected_tensor, atol=1e-4))
-
-    # XXX: this might be a candidate for common tests if we have many of those
-    def test_lm_head_ignore_keys(self):
```

Comment:
This was testing the hack added to remove the weights from the `_keys_to_ignore_on_save` when untied.

Reply:
Nice to see it go